snippets/language-ruby.cson

```diff
@@ -15,6 +15,9 @@
   'case … end':
     'prefix': 'case'
     'body': 'case ${1:object}\nwhen ${2:condition}\n\t$0\nend'
+  'begin … rescue … end':
+    'prefix': 'bre'
```

Comment:
Yeah, the non-defaults tend to follow that pattern, but the basic flow controls seem to be more consistent with the first 3 letters.